.github/scripts/generate_binary_build_matrix.py

```diff
@@ -367,5 +367,6 @@ def generate_wheels_matrix(
     return ret
 
 
+validate_nccl_dep_consistency("12.2")
```

Comment:
Is it a typo or it should be indeed `12.2`?

Reply:
Good catch Aidyn, it is a typo, thank you!

Reply:
Done.